src/backend/utils/html/sanitize.js

```diff
@@ -40,19 +38,23 @@ module.exports = function (dirty) {
       'thread',
       'tr',
       'ul',
+      'MDXPageBase',
```

Comment:
This is a losing battle.  There's no way that we can add every custom element that people will blog about, so no matter what we do, this will break when a new one pops up.

Reply:
The code actually works without adding `MDXPageBase` into the list, however we still have the issues of the new package that's still there.

I think we should change our plan of attack, and instead create a custom tag that the sanitizer will ignore. 

The past week I've been experimenting with the old `sanitize-html` package, and I found no probable way of fixing the About Us page without heavily changing the behaviour of how the sanitizer parsers our code (which I don't want to do since it's dangerous).

Should I bring this up on our meeting on Tuesday?